src/RunProcessAsTask.Tests/RunProcessAsTask.Tests.csproj

```diff
@@ -1,16 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
   <PropertyGroup>
     <TargetFramework>netcoreapp1.1</TargetFramework>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
   </PropertyGroup>
-
+  
+  
   <ItemGroup>
-    <PackageReference Include="xunit" Version="2.2.0" />
+    <Compile Include="Properties\AssemblyInfo.cs" />

```

Comment:
Does this actually work?  Aren't you getting a duplicate-entry error or warning?

It should be redundant, anyhow...